### PR TITLE
lvrt-crash-dumps: document DisableCoreDumps token (& docs enhancements)

### PR DESCRIPTION
**Pull request by amstewart**
This PR documents the new `[LVRT]DisableCoreDumps` token in `ni-rt.ini` within the "LabVIEW Core Dumps on Linux Real Time" tutorial page. This implies:
* removing previous assertions from the tutorial that LVRT doesn't keep core dumps by default.
* removing steps asking the user to modify the `lvrt-wrapper` initscript directly.
* adding a new section to the end of the tutorial, detailing how to *disable* core dumps, if the user desires.

image

White I was making changes, I also fixed up the whitespace in the document (see commit logs), and added some cute unicode ballot boxes to the MAX checkbox literals.

image

# Maintainership
* Do not merge this PR until the backing code change to support the new INI token has been merged in perforce.

### CODE REVIEW

File: docs/source/troubleshooting/lvrt-crash-dumps.rst
Comment (SparkingSpork) [inline docs/source/troubleshooting/lvrt-crash-dumps.rst:156]: Since this was technically added in lvrt and not nilrt, do we want to use the lvrt version here? I'm fine either way.

File: docs/source/troubleshooting/lvrt-crash-dumps.rst
Comment (SparkingSpork) [inline docs/source/troubleshooting/lvrt-crash-dumps.rst:70]: I would prefer we keep the documentation for "older versions" as well for now as support is still probably going to want to use this document for older versions for quite a while.

**Review by SparkingSpork — CHANGES_REQUESTED** [review]
I'd like to keep the old method for enabling core dumps here. We can tag it as outdated somehow if that's preferable but the information is still useful.

File: docs/source/troubleshooting/lvrt-crash-dumps.rst
Comment (amstewart) [reply docs/source/troubleshooting/lvrt-crash-dumps.rst:156]: Yeah; it probably makes sense to note the version of the `ni-labview-realtime` IPK. Especially since I think this behavior will differ depending on what version of the LV feeds the target is configured to use.

Will fix.

File: docs/source/troubleshooting/lvrt-crash-dumps.rst
Comment (amstewart) [reply docs/source/troubleshooting/lvrt-crash-dumps.rst:70]: Agreed. I'll add the old workflow back with a note that it only applies to older LVRT configurations.

**Comment by amstewart** [comment]
# Patch V3
* Retained the *old* workflow to modify `lvrt-wrapper`, since users might still find that relevant - especially if they're on an ARM system. @SparkingSpork 
* Used the LVRT IPK version where appropriate, rather than the NILRT version. @SparkingSpork 
* Rewrote the workflow to match with the *latest* version of the LVRT IPK and NILRT x64 changes.

**Major Change Areas**
image

image

**Review by SparkingSpork — APPROVED** [review]

**Review by SparkingSpork — COMMENTED** [review]
I've approved this but will hold off on pulling the changes until the changes internally are approved as well.

**Pull request merged**
